src/app/pages/Sandbox/Editor/Content/Header/ShareView.js

```diff
@@ -296,11 +343,25 @@ class ShareView extends React.PureComponent {
                       setValue={this.setHideNavigation}
                     />
                     <PaddedPreference
+                      title="Single file entry script"
```

Comment:
**so what do we do now?**

- we let the above simplification sink in
- remove the new switch *and the switch from the address bar*
- i'll try to figure out how to not put the moduleId in the URL if it's not manually selected

That's it. What do u think?